src/port1.0/portstartupitem.tcl

```diff
@@ -660,12 +373,7 @@ proc portstartupitem::startupitem_create {args} {
     if {${startupitem.type} eq "default" || ${startupitem.type} eq ""} {
         switch -exact ${os.platform} {
             darwin {
-                set haveLaunchd ${portutil::autoconf::have_launchd}
```

Comment:
I guess this means we can also remove the check from configure and the variable in `src/port1.0/port_autoconf.tcl.in`.